app-shells/scsh/scsh-0.6.7-r3.ebuild

```diff
@@ -13,35 +13,44 @@ SRC_URI="ftp://ftp.scsh.net/pub/scsh/${MY_PV}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="amd64 ppc sparc x86"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86"
 
-DEPEND="!dev-scheme/scheme48"
+DEPEND="
+	!dev-scheme/scheme48
+	virtual/libcrypt:=
```

Comment:
Speaking of this libcrypt dependency, could you remind me again why it's needed? I grepped the source code for `crypt` but found nothing. I ran scanelf on every executable but didn't see a dependency for libcrypt.

Reply:
It links to `crypt`
```
/srfi/srfi-27.o c/unix/misc.o c/unix/io.o c/unix/fd-io.o c/unix/event.o         \
        /tmp/s48_external_$$.c                                  \
        c/unix/socket.o c/unix/dynamo.o                         \
         -lcrypt -ldl -lm -lc -lutil && 
```
Also, if this is removed portage complains:
```
* QA Notice: Binary files linked to libcrypt.so found
 * But a dependency on virtual/libcrypt is not declared!
 *
 *   /var/tmp/portage/app-shells/scsh-0.6.7-r3/image/usr/lib/scsh/scshvm
 *
 * Please add virtual/libcrypt:= dependency
 ```

Reply:
In 0.6.7 (already in tree), I see e.g.:
```
scsh/syscalls1.c:1109:/* crypt()
scsh/syscalls1.c:1112:s48_value scm_crypt(s48_value key, s48_value salt)
scsh/syscalls1.c:1114:  char * ret = crypt (s48_extract_string (key),
scsh/syscalls1.c:1209:  S48_EXPORT_FUNCTION(scm_crypt);
```

so it seems correct.

Reply:
You're right, 0.6.7 has it. I only checked the source for 0.7 and that one doesn't have any references to crypt.